src/librustc/sir.rs

```diff
@@ -55,6 +58,8 @@ pub struct SirCx {
     pub llvm_blocks: FxHashMap<*const BasicBlock, (SirFuncIdx, SirBlockIdx)>,
     /// Function store. Also owns the blocks
     pub funcs: IndexVec<SirFuncIdx, ykpack::Body>,
+    /// Stores the current position (BasicBlock, Instruction index) for each builder.
```

Comment:
I think we can shorten this to something like:
> Mirrors the insertion point for each LLVM IrBuilder

There's probably no need to re-state the types in the comment.